pkg/daemon/ceph/client/upgrade.go

```diff
@@ -131,6 +137,7 @@ func EnableReleaseOSDFunctionality(context *clusterd.Context, clusterInfo *Clust
 
 // OkToStop determines if it's ok to stop an upgrade
 func OkToStop(context *clusterd.Context, clusterInfo *ClusterInfo, deployment, daemonType, daemonName string) error {
+	okToStopRetries, okToStopDelay := getRetryConfig(clusterInfo, daemonType)
```

Comment:
Can you move this line inside the switch statement? Or will it be used for daemons other than OSDs?

Reply:
This will be used by all the daemons going through `ok-to-stop`. So kept outside the switch statement.  The `getRetryConfig` method returns the default values in cases other than `OSD` and `MDS` daemons.